README.md

```diff
@@ -167,6 +167,21 @@ The function documentation can be accessed via [pkg.go.dev](https://pkg.go.dev/g
    resources, err := ParseKubernetesYaml(values)
    ```
 
+8. To get resources from Git repositories
+   ```go
+   // Parse the repo url
+   gitUrl, err := util.ParseGitUrl(url)
```

Comment:
Specify the expected format of the  `url` arg.  Since it can vary depending on what repo type, we should be clear as to what formats we support

Reply:
Additionally, ensure this information gets added to the API description as well.